dashboard/config/scripts/AP_practice_Qs_rqb_86_CSP_4_1_1.multi

```diff
@@ -0,0 +1,11 @@
+name 'AP_practice_Qs_rqb_86_CSP_4_1_1'
+wrong '<img src="https://codehs.com/uploads/c7d593125a90ac3c790a78d5b5d098af" height="200">'
+wrong '<img src="https://codehs.com/uploads/f9506ef1e6ea5e845699feaac9686a5d" height="200">'
+wrong '<img src="https://codehs.com/uploads/2ffc72ca9ba682bcd443104de50ac8a5" height="200">'
+right '<img src="https://codehs.com/uploads/6b450add4f3ff7310fa1a3c2d6a1b27a" height="200">'
```

Comment:
Can we re-host the images on images.code.org?  Our network requirements only tell administrators to unblock `*.code.org` and everything should work.

Reply:
@bakerfranke how many of these include images? Can we ask CodeHS for a dump of them, or are we going to need to pull urls from the JSON?